tests/callistocore/test_delivery.py

```diff
@@ -48,6 +51,18 @@ def test_pdf_full_report_is_generated(self):
         self.assertIn("test answer", pdfReader.getPage(1).extractText())
         self.assertIn("answer to 2nd question", pdfReader.getPage(1).extractText())
 
+    def test_pdf_report_generated_with_timestamp(self):
+        # test_tzname matches TIME_ZONE in tests/settings.py
```

Comment:
does `override_settings` still not work now that we're pulling a string from settings?